systemtests/src/test/java/io/enmasse/systemtest/web/WebConsoleTest.java

```diff
@@ -303,6 +304,46 @@ public void doTestSortConnectionsByHostname() throws Exception {
         stopClients(clients);
     }
 
+    public void doTestMessagesMetrics() throws Exception {
+        int msgCount = 19;
+        Destination dest = Destination.queue("queue-via-web");
+        consoleWebPage.createAddressWebConsole(dest);
+        consoleWebPage.openAddressesPageWebConsole();
+
+        AmqpClient client = amqpClientFactory.createQueueClient(defaultAddressSpace);
+        client.getConnectOptions().setUsername(username).setPassword(password);
+        List<String> msgBatch = TestUtils.generateMessages(msgCount);
+
+        int sent = client.sendMessages(dest.getAddress(), msgBatch, 1, TimeUnit.MINUTES).get(1, TimeUnit.MINUTES);
+        Thread.sleep(5000);
```

Comment:
This sleep could be replace by some wait until expected count of messages/senders/receivers/...etc within some timeout (i.e 5sec)

Reply:
resolved